README.md

```diff
@@ -144,6 +144,75 @@ This section applies to Solidity and JavaScript projects alike.
 
 * Format your code using `rustfmt`.
 
+## Pull Requests
+
+_The following assumes that code and other work will continue to be stored on GitHub and that ZenHub will be the management tool_.
+
+A Pull Request (PR), which is a request for changing content within a repository, can be opened at anytime.
+
+Ideally, a PR is small, limited to a single objective, and in response to a specific GitHub issue.
+
+The PR is the only place any team member is expected to consult for information about the PR or its lifecycle.
+
+### Summary
+
+* a PR is its source of truth
+* a PR should be connected to the relevant issue(s)
+* the reviewer who takes up review formally as an item of work should self-assign on the PR
+* when the review ends, because the assignee either completes the review or will not complete the review, assignment should be cleared
+* comments on the PR should be limited to the PR
+* comments about the PR should only be made on the PR
+
+### Format
+
+The PR should be connected to any issues to which it is a response. If the PR resolves an issue, that should be indicated in the body of the PR (e.g., "Resolves #70", "Fixes #71", etc.).
```

Comment:
Should we explicitly mention that we mean "connected using ZenHub"? Otherwise, it may be unclear what "connected" means.

Reply:
@schemar: L149 notes the assumption that GitHub and ZenHub are used. However, it doesn't hurt to be explicit; I will update.